### DISCUSSION TITLE
First and Last were removed after Popup with the other slider was opened

### DISCUSSION THREAD

**Discussion by tanyasfy** [simeydotme/svelte-range-slider-pips#144] [Q&A]
I have two sliders. One is on the 'main' page, the other is in the popup. If I don't open the popup, the main slider works as needed. If I open the popup with the other slider, the first and last are removed from both sliders.
I think first and last are not reaktiv. What should I do to avoid this problem?

**Comment by simeydotme** [comment]
hi @tanyasfy , the pips/labels are handled with CSS.
If you have the CSS for the popup slider hiding the first/last pips, then you will need to use a `id=` and css like `#my-slider .last` or whatever to hide, so that it doesn't affect _ALL_ sliders.

You could also post a very simply demonstration using the REPL provided in the CONTRIBUTING.md so that I can help you debug the issue.

Thanks!
